deploy/restartComponent.sh

```diff
@@ -22,9 +23,8 @@ for comp in $COMPONENTS; do
       exit 1
     fi
 
-    . /data/admin/wmagent/env.sh
     TAIL_LOG=`tail -n100 $COMPLOG`
-    $WMA_PATH/manage execute-agent wmcoreD --restart --components=$comp
+    $manage execute-agent wmcoreD --restart --components=$comp
```

Comment:
This `$manage` call should be executing the correct manage script which comes with the current agent setup and should be set from the agent's environment (and not that we are not used to it), but it is actually dangerous to call it directly without checking if it does exist and where it leads to. So the previously full path referring to the executable seemed better to me. The alternative would be to check if the variable is set and leads to something expected...... something in the sense of: 

```
[[ -x $manage ]] && [[ $WMA_PATH/manage ]] || { echo "Either not set or not executable '$manage' script." ; exit 1 ;}  
$manage execute-agent wmcoreD --restart --components=$comp
```

Reply:
I think that would be overdoing these changes. Whenever we connect to an agent, we source the env.sh script and we trigger $manage commands, without checking anything. 
I agree that such checks make the code more robust, but there is always extra lines and extra complication that might not be necessary, which is this case IMO.